web/src/navbar_alerts.ts

```diff
@@ -420,6 +464,35 @@ const time_zone_update_offer_banner = (): AlertBanner => {
     };
 };
 
+const upload_file_quota_warning_banner = (quota: number): AlertBanner => {
+    const is_limited_plan = realm.realm_plan_type === settings_config.realm_plan_types.limited.code;
+    return {
+        process: "upload_quota_warning",
+        intent: "warning",
+        label: $t(
+            {
+                defaultMessage:
+                    "Your organization is using {percentage_upload_space}% of your {upload_quota}file storage quota.{upgrade_message}",
+            },
```

Comment:
I think declaring the banner separately for admins and non-admins would make the code more readable, similar to say `bankruptcy_banner()` and `demo_organization_deadline_banner`.